### PR TITLE
INTGR-166: "Add-Ins for Data Storage" page

### PR DESCRIPTION
**Pull request by tranjennie**

### CODE REVIEW

File: src/pages/myGeotab/addIns/addInStorage.tsx [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+                </div>
+                <h1 className="title">Add-In for Data Storage</h1>
+            </div>
+            <div className="paragraph">
+                <p>This page provides an overview of the storage (AddInData) API and describes its use within Add-Ins.</p>
+                <InformationalBox>
+                    <p>For a sample add-in that uses the storage API go to this <a href="https://github.com/Geotab/sdk-addin-samples/tree/master/storage-api-sample">link</a>.</p>
+                </InformationalBox>
+            </div>
+            <Accordion summary="What is the Storage API?" p={StorageAPI}></Accordion>
```
Comment (kmets89) [inline src/pages/myGeotab/addIns/addInStorage.tsx:493]: please capitalize all words in all accordion titles

File: src/pages/myGeotab/addIns/addInStorage.tsx [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+        <ol>
+            <li><strong>Administrator, Supervisor, Default User, Drive App User</strong> {`=>`} "Add/Set/Get/Remove"</li>
+            <li><strong>ViewOnly</strong> {`=>`} "Get"</li>
+            <li><strong>Nothing</strong> {`=>`} None</li>
+        </ol>
+
+        <h3>Small vs Large</h3>
+        <p>While it's possible to create a single AddInData object with an array of details, this approach is less scalable. First is contending with the mandatory limit of 10,000 characters. Second is that it can cause unduly large objects to deal with which can be less memory efficient. Third is that if there is an array of entries and you need to remove one, you will have to remove the whole object and add a new one with the updated list of details. In general, we have found it more useful to treat the AddInData as a simple object which there can be many of.</p>
+    </div>
+    <h2>Additional Resources</h2>
```
Comment (kmets89) [inline src/pages/myGeotab/addIns/addInStorage.tsx:463]: according to current page markup this should be its own section/accordion

File: src/pages/myGeotab/addIns/addInStorage.tsx [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+    <ol>
+        <li>The JSON data for an AddInData object must be 10,000 characters or less.</li>
+        <li>No property in the JSON data can have the format "geotabXYZ". This naming format is reserved for Geotab use.</li>
+    </ol>
+</div>;
+
+
+const AdditionalNotes: ReactNode = <div className="paragraph">
+    <div className="paragraph">
+        <h3>Legacy Property 'Data'</h3>
```
Comment (kmets89) [inline src/pages/myGeotab/addIns/addInStorage.tsx:421]: all ### from markup should be h2, not h3

File: src/pages/myGeotab/addIns/addInStorage.tsx [hunk 4]
```diff
@@ -0,0 +1,502 @@
...
+    }],
+    "customer": {
+        "name": "joesmith",
+        "email": "joe@smith.com"
+    }
+}`} />
+</div>;
+
+//
+const CreatingAddInDataObject: ReactNode = <div className="paragraph">
```
Comment (kmets89) [inline src/pages/myGeotab/addIns/addInStorage.tsx:36]: use camel case for these variables

**Review by kmets89 — CHANGES_REQUESTED** [review]

**Comment by tranjennie** [comment]
Hello @kmets89 !
made some changes: 
modify heading levels from h3 to h2, change titled, added TODO comments

**Pull request merged**